### PR TITLE
Bump Psammead media player and add correct bottom padding to Figure

### LINKED ISSUE #7692: Bump psammead-media-player and include custom styles
**Is your feature request related to a problem? Please describe.**
Psammead-media-player has had its bottom margin removed and this should be set within simorgh where the component is used as different pages have different styling requirements for the media player

**Describe the solution you'd like**
- Bump psammead-media-player
- Remove the bottom padding override for the figure component
- Ensure the cps media player has correct bottom margin (This is different for MAP's vs STY's or Articles)
- Ensure captions render right under the media player, they currently have some spacing above them

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves https://github.com/bbc/simorgh/issues/7692

**Overall change:**
Pull in the latest psammead media player and ensure the figure component has the correct bottom padding

**Code changes:**

- Bumps psammead media player to the latest version
- Ensures the correct bottom padding is applied to the Figure component
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
         "debug": "^4.1.0",
         "gensync": "^1.0.0-beta.1",
         "json5": "^2.1.2",
-        "lodash": "^4.17.20",
+        "lodash": "^4.17.19",
```
Comment (karinathomasbbc) [inline package-lock.json]: Uurgh this again!!

**Review by karinathomasbbc — APPROVED** [review]
Looks good - may need to revert some of the package-lock changes 👍

File: package-lock.json (hunk 1 above)
Comment (chris-hinds) [reply package-lock.json]: Ok now I am not actually sure why this is happening :(

Comment (chris-hinds) [reply package-lock.json]: This looks to have been fixed now

**Review by pharingee — APPROVED** [review]
LG

**Comment by RichardPK** [comment]
Going to propagate these changes to the OD TV Page & props to the AVPlayer - please don't merge 'til then 👍 

_Edit: These changes are in_

**Comment by RichardPK** [comment]
@pharingee @karinathomasbbc I've just updated this to include the changes for OD TV, can you sense-check and then I think we're good to merge?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,6 @@
   "version": "1.0.0",
   "description": "Simorgh",
   "main": "./src/client.js",
-  "resolutions": {},
```
Comment (karinathomasbbc) [inline package.json]: Is this deletion intentional?

File: src/app/containers/AVPlayer/index.jsx [hunk 3]
```diff
@@ -8,6 +14,15 @@ import pathOr from 'ramda/src/pathOr';
 import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
 
+const Wrapper = styled.div`
```
Comment (karinathomasbbc) [inline src/app/containers/AVPlayer/index.jsx:22]: Could we call this `StyledAVPlayer` instead of `Wrapper`?

File: package.json (hunk 2 above)
Comment (chris-hinds) [reply package.json]: it is yes.

**Review by karinathomasbbc — APPROVED** [review]
👍

File: src/app/containers/AVPlayer/index.jsx (hunk 3 above)
Comment (RichardPK) [reply src/app/containers/AVPlayer/index.jsx:22]: So at the OnDemandTvPage level (above the wrapper) there is already a `StyledVideoPlayer` (which is a styled version of AVPlayer), and in the psammead-media-player (level below) there's a `StyledVideoContainer`... I dunno I just find all the repetition of `Styled` & repeated 'player/video' declarations a bit confusing, and in this instance - it isn't the AV player. It's a div that wraps the AV player.

We don't seem to have a consistent approach to this though, so 🤷‍♂️

Comment (karinathomasbbc) [reply src/app/containers/AVPlayer/index.jsx:22]: Fair enough, let's leave it as is

**Pull request merged**